src/OpenAL/OpenToolkit.OpenAL/ALTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+    internal class ALTest
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Hello!");
+            var device = ALC.OpenDevice(string.Empty);
+            var context = ALC.CreateContext(device, (int[])null);
+            ALC.MakeContextCurrent(context);
+
+            string alcExts = ALC.GetString(device, AlcGetString.Extensions);
+
+            string exts = AL.Get(ALGetString.Extensions);
+            string rend = AL.Get(ALGetString.Renderer);
+            string vend = AL.Get(ALGetString.Vendor);
+            string vers = AL.Get(ALGetString.Version);
```

Comment:
Please write out the variable names. And use var as well?

Reply:
This is just a temporary file that will be removed before merge

Reply:
It could still live as a Test application somewhere in `/Tests` as part of the development environment in a separate project maybe, which is not part of the nupkeg later.